app/services/users.js

```diff
@@ -1,14 +1,28 @@
 const { User } = require('../models');
 const logger = require('../logger/index');
 const errors = require('../errors');
+const { formatUserOutput } = require('../serializers/users');
 
 exports.createUser = async payload => {
   try {
     const userCreated = await User.create(payload);
 
-    return userCreated;
+    return formatUserOutput(userCreated);
```

Comment:
put this in the controller, try to leave any logic outside of the service functions